src/client/cli/formatter/table_formatter.cpp

```diff
@@ -53,91 +53,150 @@ std::string to_usage(const std::string& usage, const std::string& total)
     return fmt::format("{} out of {}", mp::MemorySize{usage}.human_readable(), mp::MemorySize{total}.human_readable());
 }
 
-std::string generate_instance_info_report(const mp::InfoReply& reply)
+std::string generate_snapshot_details(const mp::DetailedInfoItem& item)
 {
     fmt::memory_buffer buf;
+    const auto& fundamentals = item.snapshot_info().fundamentals();
+
+    fmt::format_to(std::back_inserter(buf), "{:<16}{}\n", "Snapshot:", fundamentals.snapshot_name());
+    fmt::format_to(std::back_inserter(buf), "{:<16}{}\n", "Instance:", item.name());
+    fmt::format_to(std::back_inserter(buf), "{:<16}{}\n",
+                   "Size:", item.snapshot_info().size().empty() ? "--" : item.snapshot_info().size());
+    fmt::format_to(std::back_inserter(buf), "{:<16}{}\n", "CPU(s):", item.cpu_count());
+    fmt::format_to(std::back_inserter(buf), "{:<16}{}\n", "Disk space:", item.disk_total());
+    fmt::format_to(std::back_inserter(buf), "{:<16}{}\n", "Memory size:", item.memory_total());
+
+    auto mount_paths = item.mount_info().mount_paths();
+    fmt::format_to(std::back_inserter(buf), "{:<16}{}", "Mounts:", mount_paths.empty() ? "--\n" : "");
+    for (auto mount = mount_paths.cbegin(); mount != mount_paths.cend(); ++mount)
+    {
+        if (mount != mount_paths.cbegin())
+            fmt::format_to(std::back_inserter(buf), "{:<16}", "");
+        fmt::format_to(std::back_inserter(buf), "{:{}} => {}\n", mount->source_path(),
+                       item.mount_info().longest_path_len(), mount->target_path());
+    }
```

Comment:
Having looked at it, I wonder if we should stick to the current format... WDYT?

Reply:
The part that is omitted is the id mappings, correct? Personally, I think its fine if they're not shown in snapshot info as I would expect those finer details to only become relevant when inspecting the current state of an instance. Additionally, including id mappings will increase the relative length of the `mounts` section to everything else which has more of a terse feel to it.
These are my justifications for it, but I'm not opposed to including more details as `info` is supposed to be giving the user detailed information and they should be using `snapshot-overview` to get less details.

Reply:
OK, thanks. Let's leave it then.